components/Card.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <NuxtLink :to="`/products/${items.id}`">
+    <UCard class="w-[100%] max-h-[400px] overflow-hidden" @click="handleCardClick(items.id)">
+      <template #header>
+        <div>
+          <img :src="items.images[0]" alt="" class="w-[100%] h-[300px] object-cover rounded-t-lg" />
```

Comment:
Optional also works on array indexing. `?.[0]`

Reply:
@archit-credilio I have resolved the comment